fix: update some Django alt text (#21773)
mdn/content: files/en-us/learn/server-side/django/deployment/index.md

All projects in the Github repos you shared with Railway during setup are displayed.
Select your Github repository for the local library: `<user-name>/django-locallibrary-tutorial`.

> ![Railway website screen showing a dialog to choose and existing GitHub repository or chose a new one](railway_new_project_button_deploy_github_selectrepo.png)

Reviewer: ```suggestion
![Railway website screen showing a dialog to choose an existing GitHub repository or choose a new one](railway_new_project_button_deploy_github_selectrepo.png)
```
